### ISSUE TITLE
Is this project dead or unmaitained?

### ISSUE THREAD

**Issue by alienbogart** [aniketpanjwani/chomper#49]
I posted an issue in[ January 6](https://github.com/aniketpanjwani/chomper/issues/48)  and still got no response. Is there anyone working on this project at all? Thanks.

**Comment by ghost** [comment]
I'm also interested in the answer.

**Comment by aniketpanjwani** [comment]
Hi folks,

I am very sorry for the long delay in getting back to you. The delay has been due to some part being busy with work/life, and some part procrastination.

One problem which stalled development is that I didn't have a very good idea of how to test the behavior of Chomper. I can't easily create and run tests locally, even inside a Docker container or a VM, because (I think) a VM will share some of the network configuration of its host (like the hosts file), and Chomper may require messing around with these root-protected files.

However, I think that I can properly test Chomper by spinning up a temporary VPS, using a cloud service like AWS EC2. 

I am going to begin rationing out a few hours each week to Chomper, starting tomorrow. My first priority will be developing tests, and then hopefully, I will eventually get to the large list of issues.

Thanks!

Aniket

**Comment by ghost** [comment]
thanks!

**Comment by ghost** [comment]
how is it going? 
sorry for being too direct, but I wanna keep you accountable :)

**Comment by alienbogart** [comment]
Yeah, not to be judgemental or anything, this is FOSS after all, but it would be awesome to see this project up and running again. Cheers!

**Comment by SamLawAnthro** [comment]
Any updates? is it really dead now? I'd love to help but I'm probably not technically savvy enough to deal with bigger problems.

**Comment by phanirithvij** [comment]
Anyone know of any alternatives?
